src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  data: [],
+  status: 'idle',
+  error: null,
+};
+
+export const fetchRocketsData = createAsyncThunk('rockets/fetchData', async () => {
+  try {
+    const response = await axios.get('https://api.spacexdata.com/v3/rockets');
+    return response.data;
```

Comment:
As per the [requirement ](https://github.com/DevaWinner/space-travelers-hub/issues/17), we are to dispatch an action to store the selected data in Redux store once the data are fetched, 

The required data are:
- id
- name
- type
- flickr_images

The current implementation fetches and stores the entire rocket data from the API. It is advisable to explicitly filter and store only the required fields (id, name, type, flickr_images) in the Redux store to reduce unnecessary data and improve performance.

You can consider implementing it this way instead

```suggestion
    const response = await axios.get('https://api.spacexdata.com/v3/rockets');
    const rocketsData = response.data.map((rocket) => ({
      id: rocket.id,
      name: rocket.name,
      type: rocket.type,
      flickr_images: rocket.flickr_images,
    }));
    return rocketsData;
```

If you have any question, feel free to ask me.